### PR TITLE
Feature/pager

### PR DESCRIPTION
**Pull request by fraincs**
- Styled Pager according to new brand design.

Ticket: IGL-74

### CODE REVIEW

File: packages/Pager/src/pager.scss
Comment (franckgaudin) [inline packages/Pager/src/pager.scss:1]: The focus state lacks a background. If you look in Figma, you'll see that it's the same as the hover state.

pager-focus-state

File: packages/Pager/src/pager.scss [hunk 1]
```diff
@@ -1,192 +1,292 @@
...
+    --ids-pager-item-wrapper-border-size: 0.0625rem;
+    --ids-pager-item-wrapper-padding: calc(var(--hop-space-inset-xs) - var(--ids-pager-item-wrapper-border-size));
+    --ids-pager-item-wrapper-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-padding: 0;
+    --ids-pager-item-background: var(--hop-neutral-surface);
+    --ids-pager-item-border: none;
+    --ids-pager-item-prev-border-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-next-border-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-color: var(--hop-neutral-text);
+    --ids-pager-item-radius: 999px;
```
Comment (franckgaudin) [inline packages/Pager/src/pager.scss:78]: Use the hopper tokens 
```suggestion
    --ids-pager-item-radius: var(--hop-shape-circle);
```

File: packages/Pager/src/pager.scss [hunk 2]
```diff
@@ -1,192 +1,292 @@
...
+    --ids-pager-item-selected-background: var(--hop-neutral-surface-strong);
+    --ids-pager-item-selected-color: var(--hop-neutral-text-strong);
+
+    /* Hover */
+    --ids-pager-item-hover-border: none;
+    --ids-pager-item-hover-background: var(--hop-neutral-surface-hover);
+
+    /* Focus */
+    --ids-pager-item-focus-border-width: 0;
+    --ids-pager-item-focus-outline-width: 0.09375rem;
```
Comment (franckgaudin) [inline packages/Pager/src/pager.scss:94]: There is an eslint rule that limits the number after the decimal point, use `0.0938rem`

Comment (fraincs) [reply packages/Pager/src/pager.scss:94]: That would not equals 1.5px. Base 16 uses a lot of decimals versus base 10. As in the end screens renders px's I would rethink this rule, what is the goal of that rule?

Comment (franckgaudin) [reply packages/Pager/src/pager.scss:94]: This rule has been defined in the standards of the [wl-web-config](https://github.com/gsoft-inc/wl-web-configs) library, so you can open an issue to have it changed.

**Review by franckgaudin — APPROVED** [review]

**Review by ofrogon — APPROVED** [review]

**Pull request merged**